### DISCUSSION TITLE
Drag-and-drop implementation for reorderable arrays for MUI5

### DISCUSSION THREAD

**Discussion by Gungy2** [rjsf-team/react-jsonschema-form#3992] [Ideas]
Hello! I have created a `ArrayFieldTemplate` + `ArrayFieldItemTemplate` for MUI5 that supports drag-and-drop using [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd). I am wondering if it is something that you are interested in having in the library. Thanks!

**Comment by grutt** [comment]
Id be interested in your implementation even if it's a separate repo
